examples/pvc/download-model.md

```diff
@@ -0,0 +1,81 @@
+# Loading a model directly from a PVC
+
+Downloading large models from Hugging Face can take a significant amount of time. If a PVC containing the model files is already pre-populated, then mounting this path and supplying that to vLLM can drastically shorten the engine's warm up time.
+
+
+## 1. How to download the model onto a PVC
+
+There are some requirements for the PV or StorageClass. In particular, select a storage class that provides
+- a retention policy of `Retain`, to ensure that the model downloaded remains on the PV despite no PVCs attached
+- the desired access: **ReadWriteMany RWX** (preferred) vs. ReadWriteOnce RWO
+  - this is to ensure that at least one pod can mount to the storage volume and download the model, which can be later read by another pod
+  - RWX will also support multinode and multiple replicas, while RWO won't
+
+You should then ask your administrator for a PVC that is available in your cluster. Assuming that a RWX PVC is available, which allows many pods across nodes to read-write to the volume, you can create a pod using the following spec which downloads your desire model onto the PVC.
```

Comment:
I think this paragraph does not make sense. In a couple of lines you show how to create one. (vs asking for one).

Reply:
Better now?